public/app/features/alerting/unified/components/import-to-gma/ImportFromDSRules.tsx

```diff
@@ -0,0 +1,205 @@
+        t('alerting.import-to-gma.success', 'Successfully exported alert rules to Grafana-managed rules.')
+      );
+      locationService.push(ruleListUrl);
+    } catch (error) {
+      notifyApp.error(
+        t('alerting.import-to-gma.error', 'Failed to export alert rules: {{error}}', {
+          error: stringifyErrorLike(error),
+        })
+      );
+    }
+  };
+
+  const notifyApp = useAppNotification();
+  const [optionsShowing, toggleOptions] = useToggle(false);
+
+  return (
+    <AlertingPageWrapper
+      navId="alert-list"
+      pageNav={{
+        text: t('alerting.import-to-gma.pageTitle', 'Export alert rules from a datasource to Grafana-managed rules.'),
+      }}
+    >
+      <Stack gap={2} direction={'column'}>
+        <Text element="h2" variant="h5">
+          <Trans i18nKey="alerting.import-to-gma.title">Migrate your alert rules from a datasource into Grafana.</Trans>
+        </Text>
+        <FormProvider {...formAPI}>
+          <form onSubmit={handleSubmit(onSubmit)}>
+            <InlineField
+              transparent={true}
+              label={t('alerting.import-to-gma.datasource.label', 'Datasource')}
+              labelWidth={20}
+              invalid={!!errors.selectedDatasourceName}
+              error={errors.selectedDatasourceName?.message}
+            >
+              <Controller
+                render={({ field: { onChange, ref, ...field } }) => (
+                  <CloudRulesSourcePicker
+                    {...field}
+                    onChange={(ds: DataSourceInstanceSettings) => {
+                      setValue('selectedDatasourceUID', ds.uid);
+                      setValue('selectedDatasourceName', ds.name);
+                    }}
+                  />
+                )}
+                name="selectedDatasourceName"
+                rules={{
+                  required: {
+                    value: true,
+                    message: t('alerting.import-to-gma.datasource.required-message', 'Please select a datasource'),
+                  },
+                }}
+                control={control}
+              />
+            </InlineField>
+
+            <Collapse
+              label={t('alerting.import-to-gma.optional-settings', 'Optional settings')}
+              isOpen={optionsShowing}
+              onToggle={toggleOptions}
+              collapsible={true}
+            >
+              <Stack direction={'column'} gap={0}>
+                <InlineField
+                  transparent={true}
+                  label={t('alerting.import-to-gma.target-folder.label', 'Target Folder (optional)')}
+                  labelWidth={20}
+                  invalid={!!errors.selectedDatasourceName}
+                  error={errors.selectedDatasourceName?.message}
+                >
+                  <Controller
+                    render={({ field: { onChange, ref, ...field } }) => (
+                      <Stack width={50}>
+                        <NestedFolderPicker
```

Comment:
The picker isn't able to overflow out of the collapsing section btw:
![image](https://github.com/user-attachments/assets/53b9208b-f19c-422e-9200-e2dbbbad9a61)

Reply:
(I have a fix for this, will do in a separate PR)

Reply:
https://github.com/grafana/grafana/pull/102634 for the fix for this